dpctl/memory/_memory.pyx

```diff
@@ -751,11 +751,29 @@ def as_usm_memory(obj):
         )
 
 
-cdef api DPCTLSyclUSMRef get_usm_pointer(_Memory obj):
+cdef api DPCTLSyclUSMRef Memory_GetUsmPointer(_Memory obj):
+    "Pointer of USM allocation"
```

Comment:
Is this string here on purpose? Was it intended as a docstring?

Reply:
Yes, it was intended as a docstring, but I am not sure if this registers anywhere, seeing how this is a C function only. I suppose Cython just silently drops the docstring

Reply:
The only trace of that docstring is left in the Cython-generated `.cpp` file. For example, in master:

```
14920/* "dpctl/_sycl_queue.pyx":1004
14921 *
14922 *
14923 * cdef api DPCTLSyclQueueRef get_queue_ref(SyclQueue q):             # <<<<<<<<<<<<<<
14924 *     """
14925 *     C-API function to get opaque queue reference from
14926 */
14927
14928static DPCTLSyclQueueRef __pyx_f_5dpctl_11_sycl_queue_get_queue_ref(struct PySyclQueueObject *__pyx_v_q) {
14929  DPCTLSyclQueueRef __pyx_r;
14930  __Pyx_TraceDeclarations
```